.changeset/wild-gorillas-care.md

```diff
@@ -0,0 +1,27 @@
+---
+"@latticexyz/std-client": major
+"@latticexyz/common": major
```

Comment:
should this be a `minor` since its a new feature, not a breaking change?

Reply:
yep good catch!